core/src/main/scala/com/karumi/shot/android/Adb.scala

```diff
@@ -11,6 +11,10 @@ object Adb {
 class Adb {
 
   private final val CR_ASCII_DECIMAL = 13
+  private val logger = ProcessLogger(
+    o => println("Shot ADB output: " + o),
```

Comment:
I think this should be more descriptive as `output`

Reply:
I don't get what you suggest

Reply:
```suggestion
    output => println("Shot ADB output: " + o),
```

Reply:
Fixed!